src/scripts/remove-bloatware-apps.ps1

```diff
@@ -35,7 +35,6 @@ function Remove-BloatwareApps() {
         "Microsoft.People"                      # People
         "Microsoft.MSPaint"                     # Paint 3D (Where every artist truly start as a kid, i mean, on original Paint, not this 3D)
         "Microsoft.Print3D"                     # Print 3D
-        "Microsoft.ScreenSketch"
```

Comment:
Ok, but instead of removing the line, can you add this line below the line `113` (`#"Microsoft.FreshPaint"...`) and comment with '#'?
So other people could remove this App by just uncommenting it.

Btw, i was using Win + Shift + S to screenshot, but i didn't make a real clean install (upgraded from 10), so i didn't knew this detail.

Reply:
Done on 9e2f44b